docs/content/docs/deployment/azure-frs.md

```diff
@@ -105,22 +105,30 @@ const { audience } = containerServices;
 const audienceDiv = document.createElement("div");
 
 const onAudienceChanged = () => {
-    const members = audience.getMembers();
-    const self = audience.getMyself();
-    const memberNames = [];
-    members.forEach((member) => {
-        if (member.userId !== self?.userId) {
-            memberNames.push(member.userName);
-        }
-    });
-    audienceDiv.innerHTML = `
-        Current User: ${self?.userName} <br />
-        Other Users: ${memberNames.join(", ")}
-    `;
-};
-
-onAudienceChanged();
-audience.on("membersChanged", onAudienceChanged);
+        const members = audience.getMembers();
+        const self = audience.getMyself();
+        const memberStrings: string[] = [];
+        const useFrs = process.env.FLUID_CLIENT === "frs";
+
+        members.forEach((member: FrsMember<ICustomUserDetails>) => {
+            if (member.userId !== self?.userId) {
+                if (useFrs) {
+                    const memberString = `${member.userName}: {Email: ${member.additionalDetails?.email},
+                        Address: ${member.additionalDetails?.address}}`;
+                    memberStrings.push(memberString);
+                } else {
+                    memberStrings.push(member.userName);
+                }
+            }
+        });
+        audienceDiv.innerHTML = `
+            Current User: ${self?.userName} <br />
+            Other Users: ${memberStrings.join(", ")}
+        `;
+    };
+
+    onAudienceChanged();
+    audience.on("membersChanged", onAudienceChanged);
 ```
 
 `audience` provides two functions that will return `FrsMember` objects that have a user ID and user name:
```

Comment:
I believe we also need to update the sample `FrsMember` value below to include the additionalDetails now

Reply:
Also, we should specifically state that the values in those fields match up to the values provided in the token

Reply:
Yep! Sorry missed that